62.CAD/main.cpp

```diff
@@ -1747,7 +1789,7 @@ class CADApp : public ApplicationBase
 
 		m_Camera.setOrigin({ 0.0, 0.0 });
 		m_Camera.setAspectRatio((double)window->getWidth() / window->getHeight());
-		m_Camera.setSize(10.0);
+		m_Camera.setSize(200.0);
```

Comment:
do this only for your case. make constexpr if CASE_2 and setCameraSize as you wish